examples/.interop/next-prisma-starter/.eslintrc

```diff
@@ -13,17 +14,26 @@
   },
   "rules": {
     // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
-    "@typescript-eslint/explicit-function-return-type": "off",
-    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/consistent-type-definitions": "off",
     "react/react-in-jsx-scope": "off",
     "react/prop-types": "off",
-    "@typescript-eslint/no-explicit-any": "off"
+
+    // Consider removing these rule disables for more type safety in your app ✨
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
We right now have [`no-explicit-any`](https://typescript-eslint.io/rules/no-explicit-any/) moved into the strict config for v6. But I'm questioning that choice - it seems like a rule some folks like. Do you have feelings on it?

Reply:
We need `any`'s throughout the codebase so it should definetely be turned off